testing-unit-go/README.md

```diff
@@ -0,0 +1,24 @@
+# Unit Testing Pulumi programs in Go
+
+An example of writing mock-based unit tests with both infrastructure definition and tests written in Go.
+
+## Prerequisites
+
+[Install Go](https://golang.org/doc/install).
+
+## Running the tests
+
+1. Restore your Go dependencies. This example currently uses [Dep](https://github.com/golang/dep) to do so:
```

Comment:
This should probably be changed to use go modules. I think that work is done/nearly done now for the providers.

Reply:
FWIW - We will need to do a pass over all examples to make this transition.  I think it's worth doing that orthogonally to this PR.